llvm/include/llvm/ProfileData/InstrProfReader.h

```diff
@@ -123,6 +123,8 @@ class InstrProfReader {
 
   virtual bool instrEntryBBEnabled() const = 0;
 
+  virtual bool instrLoopEntriesEnabled() const = 0;
```

Comment:
I think it would be sensible to add a comment to this function, even when it's somewhat trivial.